app/controllers/configuration_controller.rb

```diff
@@ -724,10 +725,10 @@ def set_form_vars
       build_default_filters_tree(@edit[:current])
     when 'ui_4'
       @edit = {
-        :timeprofile => @timeprofile,
         :current     => {},
         :key         => 'config_edit__ui4',
       }
+      @edit[:timeprofile_id] = @timeprofile.id if @timeprofile
```

Comment:
@ZitaNemeckova this check should be changed to ``` @timeprofile.try(:id) ```, ```@timeprofile``` is already set before calling set_form_vars method here
https://github.com/ZitaNemeckova/manageiq/blob/50d9179f28bf9997b604662a06e69248783637fa/app/controllers/configuration_controller.rb#L361
https://github.com/ZitaNemeckova/manageiq/blob/50d9179f28bf9997b604662a06e69248783637fa/app/controllers/configuration_controller.rb#L351
https://github.com/ZitaNemeckova/manageiq/blob/50d9179f28bf9997b604662a06e69248783637fa/app/controllers/configuration_controller.rb#L470